### PR TITLE
Stub out 2022 steering election.

### PR DESCRIPTION
**Pull request by jberkus**
Also, automated replacement of some links based on old elections directory structure.

This PR sets up the base directory for the 2022 Election Officers to start doing their work.

It also does the quick-and-easy search-and-replace for `s!events/elections!elections/steering!g` across all of the steering election documents. This will not be the last link fix for these moved directories, but there is no intention to clean up all old election links.

### CODE REVIEW

**Comment by jberkus** [comment]
/sig contributor-experience
/area elections
attn @cblecker @mrbobbytables @nikhita

File: elections/steering/2020/voters.md [hunk 1]
```diff
@@ -419,7 +419,7 @@ Because of having IDs appear with different upper casing in different sources, G
 - k-toyoda-pi
 - kwiesmueller
 - lachie83
-- lappleapple
+- lorangeorange
```
Comment (cblecker) [inline elections/steering/2020/voters.md:422]: What are we doing here?

File: elections/steering/2021/voters.yaml [hunk 2]
```diff
@@ -374,7 +374,7 @@ eligible_voters:
 - kwiesmueller
 - lachie83
 - lala123912
-- LappleApple
+- LorangeApple
```
Comment (cblecker) [inline elections/steering/2021/voters.yaml:377]: Same here

**Review by cblecker — CHANGES_REQUESTED** [review]
/hold

File: elections/steering/2022/README.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
+# 2021 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
```
Comment (cblecker) [inline elections/steering/2022/README.md:1]: Can we call out that this is stubbed/draft? Or maybe just drop this file completely if its not up to date?

**Review by cblecker — APPROVED** [review]
/lgtm
/approve
/hold cancel

**Pull request merged**